parser/Builder.cc

```diff
@@ -405,6 +406,22 @@ class Builder::Impl {
         }
     }
 
+    unique_ptr<Node> call_method_missing_fun(unique_ptr<Node> receiver, const token *dot) {
+        auto dotLoc = tokLoc(dot);
+        auto level = ruby_parser::dlevel::ERROR;
+        auto err = ruby_parser::dclass::MethodWithoutSelector;
+        driver_->external_diagnostic(level, err, dotLoc.endPos(), dotLoc.endPos(), "syntax error");
```

Comment:
Is this supposed to be `dotLoc.beginPos()` `dotLoc.endPos()`, or did you mean endpos to endpos since the error should appear *after* the dot?

Reply:
Yeah it's intentional that it's zero width; that's what preserves the previous semantics of how the parse error was reported.